inc/toolbar.php

```diff
@@ -267,8 +267,8 @@ function toolbar_signature(){
     $sig = $conf['signature'];
     $sig = dformat(null,$sig);
     $sig = str_replace('@USER@',$INPUT->server->str('REMOTE_USER'),$sig);
-    $sig = str_replace('@NAME@',$INFO['userinfo']['name'],$sig);
-    $sig = str_replace('@MAIL@',$INFO['userinfo']['mail'],$sig);
+    $sig = str_replace('@NAME@',is_null($INFO) ? "" : $INFO['userinfo']['name'],$sig);
```

Comment:
```suggestion
    $sig = str_replace('@NAME@', $INFO['userinfo']['name'] ?? "" ,$sig);
```
Eventually you can use this bit simpler syntax, having the same function.

Reply:
Can confirm it fixes it as well.